src/components/FishTankSwitch.vue

```diff
@@ -161,24 +161,24 @@ export default Vue.extend({
         cursor: pointer;
       }
     }
-    .input-checkbox-wrap {
+    .inputCheckboxWrap {
       width: 33px;
       height: 18px;
       background-color:$color-gray-light;
       border-radius: 10px;
       border:1px solid $color-gray;
       float: right;
     }
-    input:disabled ~ .label-content{
+    input:disabled ~ .labelContent{
       color: $color-disabled;
       cursor: default;
     }
   }
-  .switch input:disabled + .input-checkbox-wrap{
+  .switch input:disabled + .inputCheckboxWrap{
```

Comment:
Lets see if we can use variable bindings in the template to bind a class name to a "disabled" variable that return tru or false, and triggers the class name add/remove to set the state.